src/main/java/me/earth/earthhack/impl/modules/player/ncptweaks/ListenerPosLook.java

```diff
@@ -12,22 +12,28 @@
 
 final class ListenerPosLook extends ModuleListener<NCPTweaks, PacketEvent.Receive<SPacketPlayerPosLook>> {
     public ListenerPosLook(NCPTweaks module) {
-        super(module, PacketEvent.Receive.class);
+        super(module, PacketEvent.Receive.class, SPacketPlayerPosLook.class);
     }
 
     @Override
     public void invoke(PacketEvent.Receive<SPacketPlayerPosLook> event) {
+        if (mc.player == null || mc.world == null) {
```

Comment:
Sorry, I meant putting this check inside the scheduled task. There is a small chance that between this event running on the packet thread and the scheduled task running on the main thread world or player become null, which will crash the game when the scheduled task runs.

Reply:
ah ok i didnt know that, thanks